velox/vector/arrow/Bridge.cpp

```diff
@@ -321,6 +321,12 @@ void gatherFromBuffer(
     rows.apply([&](vector_size_t i) {
       bits::setBit(dst, j++, bits::isBitSet(src, i));
     });
+  } else if (type.kind() == TypeKind::SHORT_DECIMAL) {
```

Comment:
Can we add a test for this?

Reply:
The E2EFilterTest writes Parquet via Arrow. So this path is tested there.